x-pack/platform/packages/private/upgrade-assistant/src/components/deprecation_badge.tsx

```diff
@@ -22,7 +23,7 @@ const i18nTexts = {
 };
 
 interface Props {
-  level: 'none' | 'info' | 'warning' | 'critical' | 'fetch_error';
+  level: DeprecationSeverityOrError;
```

Comment:
this had the full set of severity errors but related code only used a subset. Uncertain if it should be full set or subset